recipes/corsid/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "corsid" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9330e919c3384d1bcf86d0964d826d16f1fc3b68325b50d48e5e2bb4e1ae371f
+
+build:
+  noarch: generic
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy
+    - pandas
+    - pip
+    - pysam
+    - pytablewriter
+    - python
+  run:
+    - numpy
+    - pandas
+    - pysam
+    - pytablewriter
+    - python
+
+test:
+  imports:
+    - corsid
+
+about:
+  home: "http://github.com/elkebir-group/CORSID"
+  license: MIT
+  license_family: MIT
+  summary: "Core Sequence Identifier"
+  doc_url: 
```

Comment:
Please remove the doc_url line, since it's empty

Reply:
Fixed, thank you